src/types/data.tsx

```diff
@@ -0,0 +1,10 @@
+export interface IRecipeItem {
```

Comment:
I believe the `IInterfaceName` naming convention is only used in languages like C# and Java. The name `IRecipeItem` is probably weird for TypeScript. You could instead call it `RecipeItem` or even `Recipe`.